### PR TITLE
Myntra clone

### PR DESCRIPTION
**Pull request by sanayamahajan-23**
Closes: 512

- **Title** : Myntra-clone website 
- **Name:** : Sanaya Mahajan
- **Idenitfy yourself:** : GSSOC Contributer

### Describe the add-ons or changes you've made 📃
I have added myntra-clone folder in React-JS Projects in intermediate level

### How Has This Been Tested? ⚙️

I have run on my local computer also checked if there are changes made.

### Screenshots 📷

Screenshot from 2024-07-16 13-42-59
### Type of change ☑️

What sort of change have you made:
Added a clone of myntra website.
Example how to mark a checkbox:-

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by sanayamahajan-23** [comment]
@Avdhesh-Varshney kindly look into my pull request

**Comment by sanayamahajan-23** [comment]
@Avdhesh-Varshney please look into my pull request

File: React-JS-Projects/Intermediate/myntra-clone/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<!-- directly add the link of video (If, possible) -->
+
+[Watch the video](https://www.loom.com/share/ba686409f6da49e3b50153145fae373f?sid=9ab9667d-db35-4b31-afb8-11c0006bf10d)
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>YOUR NAME</i></b> 👦</h4>
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/myntra-clone/README.md:61]: update it

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
update the folder name to `Myntra-Clone`

**Comment by sanayamahajan-23** [comment]
did the changes as you suggested @Avdhesh-Varshney

**Comment by sanayamahajan-23** [comment]
@Avdhesh-Varshney please look into my pull request

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by Avdhesh-Varshney** [comment]
properly link your pr with assigned issue.

**Pull request merged**